sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TransactionCoordinator.java

```diff
@@ -101,4 +107,16 @@
                 }
             });
     }
+
+    public Mono<Void> dispose() {
```

Comment:
Where is this `dispose()` method called from?

Reply:
The more I think about it, this is not needed because ReactorSession.dispose() will dispose all the sendLinks.